### PR TITLE
PR review feedback for threads

### LINKED ISSUE #920: PR review feedback for threads - fix download loader, n. of threads render
Thank you Ivan for getting it up to master!

I'll merge it but there will still be some fixes to do after:
 - [ ] download percentage is not getting rendered anymore - I think
 - [ ] with a very long subject, the number of messages in thread gets hidden because it's at the end

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/pull/657#pullrequestreview-798021944_

### PR DESCRIPTION
**Pull request by ivan-ushakov**
close #920

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by ivan-ushakov** [comment]
@Kharchevskyi 
Since I'm not good in ADK but I did some changes in your code, I added you to this PR

**Comment by ivan-ushakov** [comment]
Before:

Simulator Screen Shot - iPhone 11 Pro - 2021-11-05 at 12 56 36

After:

Simulator Screen Shot - iPhone 11 Pro - 2021-11-05 at 13 25 18

File: FlowCrypt/Controllers/Inbox/InboxViewDecorator.swift [hunk 1]
```diff
@@ -30,6 +30,11 @@ extension InboxCellNode.Input {
 
         self.init(
             emailText: NSAttributedString.text(from: email, style: style, color: textColor),
+            countText: {
+                guard element.messageCount > 0 else { return nil }
+                let count = element.messageCount > 99 ? "99+" : String(element.messageCount)
+                return NSAttributedString.text(from: "(\(count))", style: style, color: textColor)
+            }(),
```
Comment (tomholub) [inline FlowCrypt/Controllers/Inbox/InboxViewDecorator.swift:37]: If it's exactly `1` then we wouldn't have to print that number. Only need to print if it's >1.

**Review by tomholub — COMMENTED** [review]
looks good - see comment

**Comment by tomholub** [comment]
But indeed the layout is somewhat broken. Anton is off work until next week. Maybe @sosnovsky or @ekievsky can help.

**Comment by sosnovsky** [comment]
I'm here, should I try to fix these layout issues?

**Comment by ivan-ushakov** [comment]
> I'm here, should I try to fix these layout issues?

Please wait for my fix and do a review

**Review by sosnovsky — APPROVED** [review]

**Comment by sosnovsky** [comment]
Looks good now 👍

**Pull request merged**